domains/melvin.is-an.app

```diff
@@ -0,0 +1,13 @@
+  "$schema": "../schemas/domain.schema.json",
+  "description": "Some unblocked games",
+  "domain": "is-an.app",
+  "subdomain": "melvin",
+  "owner": {
+    "repo": "<https://URL/to/the/repository/with/subdomain/content/sources>",
+    "email": "<no>"
+  },
+  "record": {
+    "CNAME": "melvin4life.com",
```

Comment:
Why do you want a subdomain if you already have a domain?

Reply:
Because I need different aliases for the original domain.